tools/demo_server/config/metashell.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "metashell",
+    "git": "https://github.com/sabel83/metashell.git",
+    "build": [
+      [
+        "/bin/bash",
+        "-c",
+        "git log -1 --pretty=\"%h %B\" | head -1 | sed 's/\\\\/\\\\\\\\/g' | sed 's/\"/\\\\\"/g' | sed 's/^/\"/' | sed 's/$/\\\\n This demo is supported by the Eotvos Lorand University, Faculty of Informatics\"/' > lib/core/version_desc.hpp"
```

Comment:
Or, at least it should have a different default value. what do you think?

But we could say that this is a sample config file which is also used as is by the official demo, but then we should document it in the docs.